internal/cli/template_inspect.go

```diff
@@ -0,0 +1,165 @@
+package cli
+
+import (
+	"strings"
+
+	"github.com/olekukonko/tablewriter"
+	"github.com/posener/complete"
+	"google.golang.org/protobuf/encoding/protojson"
+
+	"github.com/hashicorp/waypoint-plugin-sdk/terminal"
+	"github.com/hashicorp/waypoint/internal/clierrors"
+	"github.com/hashicorp/waypoint/internal/pkg/flag"
+	pb "github.com/hashicorp/waypoint/pkg/server/gen"
+)
+
+type ProjectTemplateInspectCommand struct {
+	*baseCommand
+
+	flagJson bool
+
+	flagID   string
+	flagName string
+}
+
+func (c *ProjectTemplateInspectCommand) Run(args []string) int {
+	flagSet := c.Flags()
+	if err := c.Init(
+		WithArgs(args),
+		WithFlags(flagSet),
+		WithNoConfig(),
+	); err != nil {
+		return 1
+	}
+	args = flagSet.Args()
+	ctx := c.Ctx
+
+	name := ""
+	if len(args) == 1 {
+		name = args[0]
+	}
```

Comment:
Should we error if length of args is more than 1? I _think_ that's common with our other commands, but I didn't check them all

Reply:
Resolved in `8329af09e`.